sos/cleaner/__init__.py

```diff
@@ -562,6 +571,31 @@ def obfuscate_report_paths(self):
                     "WARNING: binary files that potentially contain sensitive "
                     "information will NOT be removed from the final archive\n"
                 )
+            if (self.opts.treat_certificates == "obfuscate"
+                    and not is_executable("openssl")):
+                self.opts.treat_certificates = "remove"
+                self.ui_log.warning(
+                    "WARNING: No `openssl` command available. Replacing "
+                    "`--treat-certificates` from `obfuscate` to `remove`."
+                )
+            if self.opts.treat_certificates == "obfuscate":
+                self.ui_log.warning(
+                    "WARNING: certificate files that potentially contain "
+                    "sensitive information will be CONVERTED to text and "
+                    "OBFUSCATED in the final archive.\n"
+                )
+            elif self.opts.treat_certificates == "keep":
+                self.ui_log.warning(
+                    "WARNING: certificate files that potentially contain "
+                    "sensitive information will be KEPT in the final "
+                    "archive as is.\n"
+                )
+            elif self.opts.treat_certificates == "remove":
+                self.ui_log.warning(
+                    "WARNING: certificate files that potentially contain "
+                    "sensitive information will be REMOVED in the final "
+                    "archive.\n"
+                )
```

Comment:
Just a thought / option to consider: do we really want a WARNING message (either one of those three) printed on every cleaner run?